dev/system-test/firestore.ts

```diff
@@ -1070,6 +1070,31 @@ describe('Query class', () => {
       });
   });
 
+  it('supports in', () => {
+    return Promise.all([
+      randomCol.add({foo: 'bar'}),
+      randomCol.add({foo: 'baz'}),
+      randomCol.add({foo: 'boo'}),
+    ])
+      .then(() => randomCol.where('foo', 'in', ['baz']).get())
```

Comment:
This test would be a lot more interesting (and showcase the future better) if you used more than one value here.